Sources/TuistSigning/Linter/SigningLinter.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+import TuistCore
+
+protocol SigningLinting {
+    func lint(certificate: Certificate, provisioningProfile: ProvisioningProfile) -> [LintingIssue]
+}
+
+final class SigningLinter: SigningLinting {
+    func lint(certificate: Certificate, provisioningProfile: ProvisioningProfile) -> [LintingIssue] {
```

Comment:
We could add a unit test to this one to make sure we get the expected listing issue.